src/bin/upgrade/main.rs

```diff
@@ -86,6 +86,10 @@ struct Args {
     #[structopt(long = "dry-run")]
     dry_run: bool,
 
+    /// Only update a dependency if the new version is semver incompatible.
+    #[structopt(long = "skip-compatible")]
```

Comment:
Should it conflict with `to_lockfile`? Lockfile should be semver compatible with Cargo.toml, right? So this command seems to do the opposite.

Reply:
Yeah, I think that sounds reasonable.